modin/core/execution/ray/common/utils.py

```diff
@@ -34,6 +35,8 @@
 )
 from modin.error_message import ErrorMessage
 
+_OBJECT_STORE_TO_SYSTEM_MEMORY_RATIO = 0.6
```

Comment:
Why is this ratio equal to 0.6?

Reply:
It's an arbitrary number that has been around for a long time. It goes back to 3542226076bd03473b524889959bdf3b52531ee8 from 2019/01/11.

Reply:
I was just wondering why we specified it equal to this number, i.e. is there any heuristic. If someone is aware of that, it would be good to know.